cfgov/cfgov/urls.py

```diff
@@ -36,6 +37,26 @@
 )
 from regulations3k.views import redirect_eregs
 from searchgov.views import SearchView
+from v1.models import CFGOVPage
+
+
+class CFGOVSitemap(Sitemap):
+    # Set a minimum lastmod date
+    # This is to bring in changes that don't
+    # propagate via publishing (metadata updates, etc)
+    min_date = datetime(2025, 6, 4, 0, 0, 0, 0, tzinfo=timezone.utc)
```

Comment:
I don't exactly understand the logic here. Is the idea that we would change this value in code every time there's some sort of sitewide HTML change that we want to refresh the sitemap?